diff --git a/include/configs/wandboard.h b/include/configs/wandboard.h
index e42bfc5..1932629 100644
--- a/include/configs/wandboard.h
+++ b/include/configs/wandboard.h
@@ -159,4 +159,8 @@
 #define CONFIG_ENV_OFFSET		(768 * 1024)
 #define CONFIG_SYS_MMC_ENV_DEV		0
 
+#define CONFIG_SYS_REDUNDAND_ENVIRONMENT
+#define CONFIG_ENV_OFFSET_REDUND	\
+	(CONFIG_ENV_OFFSET + CONFIG_ENV_SIZE)
+
 #endif			       /* __CONFIG_H * */
